### ISSUE TITLE
nserror messages

### ISSUE THREAD

**Issue by tanner0101** [vapor/vapor#1910]
Foundation error messages are being logged as `The operation could not be completed` which is not very useful. This might be an issue with using [`LocalizedError.localizedDescription` to print the error](https://github.com/vapor/vapor/blob/3.2.2/Sources/Vapor/Logging/Logger%2BLogError.swift#L37), maybe we need to use a different property instead.

**Comment by tanner0101** [comment]
```
[ ERROR ] The operation could not be completed (NIOServer.swift:102)
[ DEBUG ] Conform `NSError` to `Debuggable` for better debug info. (NIOServer.swift:102)
```

**Comment by MrLotU** [comment]
Also see https://github.com/vapor/core/issues/178

**Comment by tanner0101** [comment]
Fixed by vapor/core#178. Thanks @MrLotU
